daemon/src/taker_cfd.rs

```diff
@@ -351,7 +353,7 @@ impl<O, M, W> Actor<O, M, W> {
 
 impl<O: 'static, M: 'static, W: 'static> Actor<O, M, W> {
     async fn handle_auto_rollover(&mut self, ctx: &mut Context<Self>) -> Result<()> {
-        tracing::debug!("Auto rollover");
+        tracing::trace!("Auto rollover message received");
```

Comment:
I _think_ we have a general `trace` log already for message that we receive.

Reply:
I'll rebase on latest master and see. I just ran it and did not see a log for these messages besides the logs I added.